float64[6] thresholds
---